# StartCorridor.srv
---
bool started
string message
